vars/openShiftUtils.groovy

```diff
@@ -163,6 +163,10 @@ def withUINode(Map parameters = [:], Closure body) {
     def jnlpLimitCpu = parameters.get('jnlpLimitCpu', "300m")
     def jnlpRequestMemory = parameters.get('jnlpRequestMemory', "256Mi")
     def jnlpLimitMemory = parameters.get('jnlpLimitMemory', "512Mi")
+    def seleniumRequestCpu = parameters.get('seleniumRequestCpu', "500m")
+    def seleniumLimitCpu = parameters.get('seleniumLimitCpu', "2")
+    def seleniumRequestMemory = parameters.get('seleniumRequestMemory', "512Mi")
```

Comment:
Would it make sense to decrease the Request/Limit Memory ratio from 6 to 3 ? I believe if we can need up to 3Gi maybe it's reasonable to give 1Gi as a minimum - I'm thinking we enforce a max ratio of 2 for memory in other clusters